src/Microsoft.AspNetCore.Mvc.Razor/project.json

```diff
@@ -25,6 +25,7 @@
     "Microsoft.AspNetCore.Mvc.Razor.Host": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.ViewFeatures": "1.0.0-*",
     "Microsoft.CodeAnalysis.CSharp": "1.3.0-*",
+    "Microsoft.DiaSymReader": "1.0.8-*",
```

Comment:
What exactly does this reference enable? In particular is the ref only req'd for one TFM?